CanvasPlusPlayground/Schema/Persistable/Quizzes/QuizType.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  QuizType.swift
+//  CanvasPlusPlayground
+//
+//  Created by Abdulaziz Albahar on 12/14/24.
+//
+
+import Foundation
+
+enum QuizType: String, Codable, CaseIterable, Identifiable {
+    var id: String { self.rawValue }
+    
+    case practiceQuiz = "practice_quiz", assignment = "assignment", gradedSurvey = "graded_survey", survey = "survey", quizzesNext = "quizzes.next", unknown = "unknown"
+    
+    var title: String {
+        switch self {
+        case .practiceQuiz: return "Practice Quizzes"
+        case .assignment: return "Assignment Quizzes"
+        case .gradedSurvey: return "Graded Surveys"
+        case .survey: return "Surveys"
+        case .quizzesNext: return "New Quizzes"
+        case .unknown: return "Unknown"
+        }
+    }
```

Comment:
nit: FYI, `switch` statements can evaluate to a value

```suggestion
    var title: String {
        return switch self {
        case .practiceQuiz: "Practice Quizzes"
        case .assignment: "Assignment Quizzes"
        case .gradedSurvey: "Graded Surveys"
        case .survey: "Surveys"
        case .quizzesNext: "New Quizzes"
        case .unknown: "Unknown"
        }
    }
```

Reply:
Actually don’t even need the `return` in this case